pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -1,86 +1,99 @@
+#!bash
```

Comment:
It doesn't bother me that much because in that context we only need internal compatibility but I understand, I'll remove it.